tests/braille-specs/de-g0.yaml

```diff
@@ -882,3 +882,9 @@ tests:
     - "-.two words'."
     - typeform:
         downgrade: "+++++++++"
+
+# If there is a single standing letter within square brackets (e.g. "[W]") directly followed by another
+# letter, Liblouis generates an unnecessary second dot 6 in front of the closing square bracket.
+  - - "[W]ährend"
+    - "'=w'=`hrend"
+    - xfail: "current output is: '=w''=`hrend"
```

Comment:
The table was updated but the test still fails?

Reply:
I know, I was wondering the same thing

Reply:
@mabo991 @egli Any idea what is going on?

Reply:
No, I have no idea